### ISSUE TITLE
Fix : Responsive for Homepage 

### ISSUE THREAD

**Issue by Surya-V-28** [N-45div/Resilient-Response#23]
I am glad work on this issues, please assign this under hacktober fest here is the sample 

for some device the UI of the home is overlapping 

![image](https://github.com/N-45div/Resilient-Response/assets/102609082/96dccb97-3ad8-4a37-ac6c-52f238e4cd89)

second set 

![image](https://github.com/N-45div/Resilient-Response/assets/102609082/cb8ede5a-2467-4d23-83d1-4bbcad145204)

please let me know if there is any issues

**Comment by N-45div** [comment]
Yep @Surya-V-28 you can carry on the work and it is accepted under hacktoberfest

**Comment by PraffulPatel27** [comment]
Hello there! 
I hope this message brings a smile to your day. 😊 If it's possible, could you please consider assigning issue #23, Responsive for Homepage, to me? I'm eager to contribute and make a positive impact on the project. Thank you for your time and consideration!

**Comment by Surya-V-28** [comment]
please @N-45div assign this to @PraffulPatel27 , for me mid sem exam going on , thanks for understanding

**Comment by PraffulPatel27** [comment]
@Surya-V-28 and  @N-45div can you tell me how can i get home page html file after clone Resilient-Response.

**Comment by N-45div** [comment]
> @Surya-V-28 and  @N-45div can you tell me how can i get home page html file after clone Resilient-Response.

It's written in react/javascript buddy go through the components of the application
